sdgym/synthesizers/column.py

```diff
@@ -17,6 +21,21 @@ class ColumnSynthesizer(BaselineSynthesizer):
     def _get_trained_synthesizer(self, real_data, metadata):
         hyper_transformer = HyperTransformer()
         hyper_transformer.detect_initial_config(real_data)
+        supported_sdtypes = hyper_transformer._get_supported_sdtypes()
+        config = {}
+        for column_name, column in metadata.columns.items():
```

Comment:
Is this going to work with the new `Metadata` object?
You might need to provide a table name.